docs/team/shawnchew.md

```diff
@@ -7,31 +7,37 @@ title: Shawn Chew's Project Portfolio Page
 
 ### Project: NUScheduler
 
-
 NUScheduler is a desktop app for managing contacts, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, NUScheduler can get your contact management tasks done faster than traditional GUI apps.
 
 Given below are my contributions to the project.
 
-* **New Feature**: To be added
+* **New Feature**: Added functionality to find `Profile`s by keywords. (Pull Request [#68](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/68))
+  * What it does: Allows the user to find existing profiles by providing a set of keywords.
+  * Justification: This feature allows user to better navigate through the profile list.
+  * Credits: This feature is adapted from code in [AddressBook Level-3](https://github.com/se-edu/addressbook-level3).
+
+* **New Feature**: Added functionality to remove `Event`s from the list. (Pull Request [#101](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/101))
+  * What it does: Allows the user to remove an existing event from the list of events.
+  * Justification: This feature allows to have a clean list of events, without all the unwanted and past events.
+  * Credits: This feature is adapted from code in [AddressBook Level-3](https://github.com/se-edu/addressbook-level3).
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=shawnchew&breakdown=true)
 
-* **Project management**:
-    * To be added
+* **Enhancements to existing features**: Refactored code base from Addressbook3 to NUScheduler. (Pull Request [#98](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/98))
+  * Justification: Ensure that method names and variables make sense in the context of the application.
 
-* **Enhancements to existing features**:
-    * To be added
+* **Enhancements to existing features**: Refactored the events to be listed in a sorted order. (Pull Request [#119](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/119))
+    * What it does: Event list displays the event in order of start time, end time, title.
+    * Justification: Allows the user to have an idea of their events in a chronological order.
 
 * **Documentation**:
     * User Guide:
-        * Added documentation for the features `profile -v` and `event -a`
+        * Added documentation for the features `profile -v` and `event -a` (Pull Request [#53](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/53))
     * Developer Guide:
-        * To be added
+        * Added implementation details for view commands. (Pull Request [#96](https://github.com/AY2223S1-CS2103T-T17-3/tp/pull/96))
 
 * **Community**:
-    * To be added
-
-* **Tools**:
-    * To be added
+    * Reviewed 15 Pull Requests on [GitHub](https://github.com/AY2223S1-CS2103T-T17-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3A%40me).
```

Comment:
Maybe you want to update this link? the `reviewed-by` is currently set to `@me`, which means it'll be set to whichever user that's logged in at the moment.

Reply:
Okay thanks!

Reply:
Hey @ShawnChew, it seems like the updated link is still incorrect, do remember to update it in the final PPP you're going to submit. Your new link just shows all the PRs excluding PRs authored by you. 

You might want to use `https://github.com/AY2223S1-CS2103T-T17-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3AShawnChew` or `https://github.com/AY2223S1-CS2103T-T17-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3AShawnChew+-author%3AShawnChew`. 

The former includes PRs authored by you while the latter does not.